test_util/test_upgrade_vpc.py

```diff
@@ -80,14 +107,57 @@ def main():
 
     # Use the CLI installer to set exhibitor_storage_backend = zookeeper.
     test_util.cluster.install_dcos(cluster, stable_installer_url, api=False)
+
+    master_list = [h.private_ip for h in cluster.masters]
+
+    cluster_api = ClusterApi(
+        'http://{ip}'.format(ip=cluster.masters[0].public_ip),
+        master_list,
+        master_list,
+        [h.private_ip for h in cluster.agents],
+        [h.private_ip for h in cluster.public_agents],
+        False,              # dns search set.
```

Comment:
Note to self: this should be moved to an optional test. No need for the entire cluster to know about that